Composer/packages/client/src/pages/language-generation/index.tsx

```diff
@@ -37,12 +38,24 @@ const LGPage: React.FC<RouteComponentProps> = props => {
   const { lgFiles, dialogs } = state;
   const [editMode, setEditMode] = useState(false);
   const [fileValid, setFileValid] = useState(true);
-  const [codeRange, setCodeRange] = useState<CodeRange | null>(null);
+  const [codeRange, setCodeRange] = useState<CodeRange | undefined>(undefined);
```

Comment:
I am new to TypeScript in React Hooks, so I am unsure if the syntax is correct. But try to minimize the code like this.

```suggestion
  const [codeRange, setCodeRange] = useState<CodeRange?>();
```